### ISSUE TITLE
mean and std Norm ?

### ISSUE THREAD

**Issue by ucas010** [lidq92/VSFA#13]
hi,dear 大佬
对于得到的视频帧数据为何进行固定mean和std参数的Norm处理啊？
codes is [here](https://github.com/lidq92/VSFA/blob/master/CNNfeatures.py#L49)
`transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])`
这个有什么参考或者说依据吗？
多谢

**Comment by lidq92** [comment]
@ucas010 这个就是预训练图像分类模型里做的预处理操作。
我直接用的预训练好的模型，所以follow这个预训练好的模型需要的预处理操作。
当然在i/vqa问题中，不做预处理操作也许更合适，但是这样需要自己预训练一个不需要预处理操作的图像分类模型。

**Comment by ucas010** [comment]
@lidq92 
多谢。
我对torch不太熟，但我发现tf训练的模型似乎都没有这个操作啊

**Comment by lidq92** [comment]
@ucas010 
早期的Caffe有减均值图像的做法，PyTorch的torchvision里提供的图像分类模型是有我代码里那个预处理操作的，可以看[torchvision/models.html#classification](https://pytorch.org/docs/stable/torchvision/models.html#classification)。
至于TensorFlow，你可以自己再好好看看相关资料，以实际训练的时候的操作为准。比如我粗略搜了一下，你可以看看这两个文件[preprocessing.py](https://github.com/tensorflow/models/blob/master/official/vision/image_classification/preprocessing.py)和[imagenet_preprocessing.py](https://github.com/tensorflow/models/blob/master/official/vision/image_classification/resnet/imagenet_preprocessing.py)。

我不使用TensorFlow，所以关于TensorFlow的具体细节你可以自己再深入看看。

祝好。
